### ISSUE TITLE
Do not include the git hash in the release name

### ISSUE THREAD

**Issue by surajssd** [dustymabe/fedpkg-kompose#3]
From next release do not include the git hash in release name.

This is wrong: :x:

```
kompose-0.4.0-0.1.git4e3300c.el7.x86_64
```

This is correct: :white_check_mark: 

```
kompose-0.4.0-0.1.el7.x86_64
```

**Comment by dustymabe** [comment]
I personally like having the git hash in there (makes it easier to `git checkout 4e3300c` in upstream) and this is the model that @ingvagabund  had me follow. I'm not sure of the technical guidelines on this but it looks like having the git hash in there might imply we are at a post release checkout of the code, which is not the case. See the examples here: https://fedoraproject.org/wiki/Package_Versioning_Examples

TLDR; if you want, we can change it. I do like having it in there, though. If we find it is technically wrong to have it in there then we'll definitely change it.

**Comment by surajssd** [comment]
@kbsingh and I were having a discussion when this came up! I think all the openshift related binaries also have moved away from naming it with git hash!

**Comment by ingvagabund** [comment]
If you always build a commit corresponding to a release, I am completely fine with this. If you pick a commit that is between releases, it is undefined what should be done. The guidelines says anytime a project has a release, you are instructed to use the release and get rid of the commit/revision string. Given most of the go packages has no release, the commit/revision is justified. However, not every go project with a releases is used properly. Usually, a project picks a commit/revision of a dependency, not a particular release. So it does not make much sense to assume releases of go projects. Only some projects have stable release which can be used as dependencies without a need to pick a commit/revision between releases.

**Comment by dustymabe** [comment]
ok @surajssd - if you want we can pull the git hash out of the release we put out today.

**Comment by dustymabe** [comment]
fixed in #5
